internal/table/table_test.go

```diff
@@ -209,7 +209,7 @@ func Test_renderRow(t *testing.T) {
 				colWidths:       []int{8, 8},
 			},
 			"  abcd    abcd  \n" +
-				"          e\n    \n" +
```

Comment:
Well, it looks like this unit-test was broken all along.  Sorry about that.

Reply:
No worries, and thank you for confirming that this was actually broken. I thought the two line breaks seemed a little suspect